src/video/emscripten/SDL_emscriptenmouse.c

```diff
@@ -209,7 +233,7 @@ Emscripten_ShowCursor(SDL_Cursor* cursor)
             curdata = (Emscripten_CursorData *) cursor->driverdata;
 
             if(curdata->system_cursor) {
-                EM_ASM_INT({
+                MAIN_THREAD_EM_ASM_INT({
```

Comment:
Switch this to `MAIN_THREAD_EM_ASM` and drop the return. (I think `EM_ASM` was less flexible when this was written.)